conf/defaults.yaml

```diff
@@ -16,16 +16,17 @@ experiment: # These are arugments specific to the experiment we are running
     root_dir: ${program.data_dir}
     seed: ${program.seed}
     batch_size: 32
-    num_workers: 4
+    num_workers: 0
 
 
 # The values here are taken from the defaults here https://pytorch-lightning.readthedocs.io/en/1.3.8/common/trainer.html#init
 # this probably should be made into a schema, e.g. as shown https://omegaconf.readthedocs.io/en/2.0_branch/structured_config.html#merging-with-other-configs
 trainer: # These are the parameters passed to the pytorch lightning Trainer object
   logger: True
-  checkpoint_callback: True
```

Comment:
Many of these defaults have been renamed. The previous values give deprecation warnings. We may want to not hardcode these values and instead let pytorch-lightning assign these values automatically.